pandas/util/testing.py

```diff
@@ -286,6 +286,14 @@ def _skip_if_mpl_1_5():
         pytest.skip("matplotlib 1.5")
 
 
+def _skip_if_mpl_1():
+    import matplotlib
```

Comment:
actually you don't need this, instead use:

``
if self.mpl_ge_2_0_0:
``

as these are defined in the base class